### PR TITLE
v1.13.3 documentation fixes

### LINKED ISSUE #4473: Confusing example on sorting columns for tabular data
The example here on Sorting by columns with Tabular data is somewhat confusing as each of the four plots look the same.

http://holoviews.org/user_guide/Tabular_Datasets.html#Sorting-by-columns

Screen Shot 2020-06-11 at 12 32 26 AM

### PR DESCRIPTION
**Pull request by philippjfr**
Fixes #4473

### CODE REVIEW

File: holoviews/plotting/bokeh/selection.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def _style_region_element(self, region_element, unselected_color):
             if 'alpha' in opt_name:
                 options[opt_name] = 1.0
 
+        print(el2_name)
```
Comment (philippjfr) [inline holoviews/plotting/bokeh/selection.py:75]:
```suggestion
```

**Comment by mmorys** [comment]
https://github.com/holoviz/holoviews/pull/4477/commits/3a10f0808fd186ed70f47bd2a02273b4de4c633e appears to cause a DeserializationError when used with [link_selections](https://github.com/holoviz/holoviews/blob/da8168a9c99373f48968408000b623cdedf1d0a0/holoviews/selection.py#L172). A new line_dash property value of `{'value': [2,4]}` is being set to the Bokeh MultiLine models via `doc.apply_json_patch`, which isn't a valid value. I couldn't trace the source of the issue, and wasn't sure if this should be submitted as an issue since it's a release candidate.

## Minimal Example
```
import numpy as np, pandas as pd, holoviews as hv, panel as pn
from holoviews.selection import link_selections
hv.extension('bokeh')

# Create dummy dataset
num_pts = 1000
df = pd.DataFrame({'x': np.linspace(0, 1, num_pts),
                   'y0': np.cumsum(np.random.randn(num_pts)),
                   'y1': np.cumsum(np.random.randn(num_pts))})
ds = hv.Dataset(df)

# Create source plot
plots = [hv.Points(ds, ['x', 'y0']), hv.Points(ds, ['y1', 'y0'])]

# Link two plots together
unlinked = hv.Layout(plots)
linked = link_selections(unlinked)

pn.Column(linked).servable()
```

## Error output
```
ERROR:bokeh.server.protocol_handler:error handling message
 message: Message 'PATCH-DOC' content: {'references': [], 'events': [{'kind': 'ModelChanged', 'model': {'id': '1078'}, 'attr': 'line_dash', 'new': {'value': [2, 4]}}, {'kind': 'ModelChanged', 'model': {'id': '1079'}, 'attr': 'line_dash', 'new': {'value': [2, 4]}}, {'kind': 'ModelChanged', 'model': {'id': '1080'}, 'attr': 'line_dash', 'new': {'value': [2, 4]}}, {'kind': 'ModelChanged', 'model': {'id': '1270'}, 'attr': 'line_dash', 'new': {'value': [2, 4]}}, {'kind': 'ModelChanged', 'model': {'id': '1271'}, 'attr': 'line_dash', 'new': {'value': [2, 4]}}, {'kind': 'ModelChanged', 'model': {'id': '1272'}, 'attr': 'line_dash', 'new': {'value': [2, 4]}}]} 
 error: DeserializationError("DashPattern couldn't deserialize {'value': [2, 4]}")
Traceback (most recent call last):
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/server/protocol_handler.py", line 90, in handle
    work = await handler(message, connection)
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/server/session.py", line 67, in _needs_document_lock_wrapper
    result = func(self, *args, **kwargs)
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/server/session.py", line 261, in _handle_patch
    message.apply_to_document(self.document, self)
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/protocol/messages/patch_doc.py", line 100, in apply_to_document
    doc._with_self_as_curdoc(lambda: doc.apply_json_patch(self.content, setter))
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/document/document.py", line 1150, in _with_self_as_curdoc
    return f()
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/protocol/messages/patch_doc.py", line 100, in <lambda>
    doc._with_self_as_curdoc(lambda: doc.apply_json_patch(self.content, setter))
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/document/document.py", line 411, in apply_json_patch
    patched_obj.set_from_json(attr, value, models=references, setter=setter)
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/core/has_props.py", line 341, in set_from_json
    descriptor.set_from_json(self, json, models, setter)
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/core/property/descriptors.py", line 611, in set_from_json
    self.property.from_json(json, models),
  File "/home/mmorys/anaconda3/envs/panel-viz-hvbeta/lib/python3.7/site-packages/bokeh/core/property/either.py", line 93, in from_json
    raise DeserializationError("%s couldn't deserialize %s" % (self, json))
bokeh.core.property.bases.DeserializationError: DashPattern couldn't deserialize {'value': [2, 4]}
```

**Comment by philippjfr** [comment]
That's really bizarre and should probably be filed as a bokeh bug if it can be reproduced.

**Comment by philippjfr** [comment]
@mattpap Any idea what could be going on here?

**Comment by mmorys** [comment]
I traced the offending line to  https://github.com/bokeh/bokeh/blob/8d8f7e57cb982f0dc179c7f47efeb02e82747a2b/bokeh/plotting/_renderer.py#L117 , specifically in `hover_glyph=make_glyph(glyphclass, kwargs, hover_visuals)`.

Minimal code for error reproduction:
```
from bokeh.models.glyphs import MultiLine
kws = {'xs': 'xs', 'ys': 'ys', 'line_alpha': 0.2, 'line_dash': 'dotted', 'line_color': 'black'}
glyph = MultiLine(**kws)
```

**Comment by philippjfr** [comment]
Thanks for tracking that down. I can't quite reproduce it based on that snippet, do you mean when you serve that on a server?

**Comment by mmorys** [comment]
Sorry, should have clarified. If you inspect the line_dash property of the resulting glyph object, it is a list [2,4] which I believe causes the error down the line. I presume it should be a string.

**Comment by philippjfr** [comment]
>Sorry, should have clarified. If you inspect the line_dash property of the resulting glyph object, it is a list [2,4] which I believe causes the error down the line. I presume it should be a string.

`line_dash` can in fact be specified as lists of integers and are seemingly converted to that in the background even when specified as a string. I'm guessing it just doesn't expect that these end up being round-tripped via the server and I'm not yet sure what's causing that either. Before we dig into this any deeper here, what's your bokeh version?

**Comment by mmorys** [comment]
bokeh 2.1.0
holoviews 1.13.3rc5 (installed by extracting source folder from github, and running `pip install .` within the folder)
python 3.7.7

**Pull request merged**